assets/models/system/roberta-large-mnli/spec.yaml

```diff
@@ -111,4 +115,20 @@ tags:
       Standard_ND96amsr_A100_v4,
       Standard_ND40rs_v2
     ]
+  finetune_compute_allow_list:
+    [
+      Standard_NV12s_v3,
+      Standard_NV24s_v3,
+      Standard_NV48s_v3,
+      Standard_NC6s_v3,
+      Standard_NC12s_v3,
+      Standard_NC24s_v3,
+      Standard_NC24rs_v3,
+      Standard_NC4as_T4_v3,
+      Standard_NC8as_T4_v3,
+      Standard_NC16as_T4_v3,
+      Standard_NC64as_T4_v3,
+      Standard_ND40rs_v2,
+      Standard_ND96asr_v4
+    ]
 version: 12
```

Comment:
Could you please change the versions of the models?

Reply:
Updated